css/style.css

```diff
@@ -0,0 +1,710 @@
+  color: #121212;
+  font-family: nyt-cheltenham, georgia, 'times new roman', times, serif;
+  font-weight: 700;
+  font-style: italic;
+  font-size: 2rem;
+  text-align: left;
+  margin-left: 0%;
+  width: 600px;
+}
+
+.footer-grid-container {
+  display: grid;
+  grid-column-gap: 10px;
+  grid-template-columns: auto auto auto auto auto auto;
+  margin-left: auto;
+  margin-right: auto;
+  width: 1180px;
+}
+
+.footer-grid-item {
+  background-color: rgba(255, 255, 255, 0.8);
+  text-align: left;
+}
+
+.footer-grid-item h3 {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.footer-grid-item ul {
+  font-weight: 400;
+}
+
+.footer-grid-item ul li {
+  float: none;
+}
+
+.footer-border-left {
+  border-left: 1px solid #e2e2e2;
+  padding-left: 11px;
+}
+
+/* .flex-container > div {
+  background-color: #f1f1f1;
+  height: 200px;
+  margin: 15px;
+} */
+
+.footer-subscription-bold-line {
+  background-color: #fff;
+  border-bottom: 1px solid #e2e2e2;
+  content: '';
+  display: block;
+  height: 17px;
+  margin-top: 0;
+  width: 171px;
+  margin-left: 0;
+  margin-right: auto;
+  color: #ebebeb;
+}
+
+.date-of-publish {
+  margin-top: 18px;
+}
+
+.date-socials {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.article-content {
+  font-size: 1.25rem;
+  line-height: 1.875rem;
+}
+
+.article-content p span {
+  bottom: 20px;
+  top: 20px;
+  text-align: center;
+  margin-left: 38%;
+  font-size: 1.1rem;
+  font-weight: 600;
+}
+
+.nav-button {
+  height: 36px;
+  margin: 1px;
+  border-radius: 4px;
+  border: 1px;
+  cursor: pointer;
+  font-family: nyt-franklin, helvetica, arial, sans-serif;
+  white-space: nowrap;
+  vertical-align: middle;
+  background-color: rgb(86, 123, 149);
+  color: rgb(255, 255, 255);
+  font-size: 13px;
+  line-height: 11px;
+  font-weight: 700;
+}
+
+.navbar-visibility-visible {
+  visibility: hidden;
+}
+
+.navbar {
+  overflow: hidden;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  background-color: #fff;
+  z-index: 1;
+}
+
+.notification {
+  font-size: 11.5px;
+  padding-left: 5px;
+}
+
+.read-comment-button {
+  float: left;
+  width: 282px;
+  margin-top: 18px;
+  height: 36px;
+  border-radius: 4px;
+  border: 1px;
+  cursor: pointer;
+  font-family: nyt-franklin, helvetica, arial, sans-serif;
+  white-space: nowrap;
+  vertical-align: middle;
+  background-color: rgb(86, 123, 149);
+  color: rgb(255, 255, 255);
+  font-size: 16px;
+  line-height: 11px;
+  font-weight: 700;
+}
+
+.img-desc {
+  margin-left: 20%;
+  margin-right: 10%;
+}
+
+.responsive-navbar-button {
+  margin-top: 12px;
+  font-size: 11px;
+  color: rgb(50, 104, 145);
+  margin-left: 8px;
+  padding-left: 8px;
+  display: block;
+  text-align: center;
+  text-decoration: none;
+}
+
+.responsive-navbar-button a {
+  color: rgb(50, 104, 145);
+}
+
+.space-image {
+  display: block;
+  margin-left: auto;
+  width: 80%;
+}
+
+.services ul {
+  font-size: 1rem;
+}
+
+.services a {
+  padding-top: 2px !important;
+}
```

Comment:
Try not to use important to override styles. Instead, try to use CSS specificity

Reply:
Override style is removed